crd-convert/src/main/java/io/strimzi/kafka/crd/convert/converter/KafkaConverter.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.crd.convert.converter;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.JsonNodeCreator;
+import io.strimzi.api.annotations.ApiVersion;
+import io.strimzi.api.kafka.model.Kafka;
+import io.strimzi.api.kafka.model.KafkaClusterSpec;
+import io.strimzi.api.kafka.model.listener.arraylistener.ArrayOrObjectKafkaListeners;
+import io.strimzi.api.kafka.model.listener.arraylistener.GenericKafkaListener;
+import io.strimzi.api.kafka.model.listener.arraylistener.GenericKafkaListenerConfiguration;
+import io.strimzi.api.kafka.model.listener.arraylistener.KafkaListenerType;
+import io.strimzi.api.kafka.model.template.ExternalServiceTemplate;
+import io.strimzi.api.kafka.model.template.ExternalTrafficPolicy;
+import io.strimzi.api.kafka.model.template.KafkaClusterTemplate;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+@SuppressWarnings("unchecked")
+public class KafkaConverter extends Converter<Kafka> {
+
+    public static final VersionConversion<Kafka> V1ALPHA1_TO_V1BETA1 = toVersionConversion(ApiVersion.V1ALPHA1, ApiVersion.V1BETA1);
+
+    public static final VersionConversion<Kafka> V1BETA1_TO_V1BETA2 = toVersionConversion(
```

Comment:
I think you are missing affinity and tolerations in `/spec/entityOperator` and `/spec/topicOperator`

Reply:
OK, added ...

Reply:
Hmmm ... there is no tolerations property on these two ...

Reply:
And there is no template/pod either ... to move it to ...

Reply:
Yeah, I noticed when putting together the document that the tolerations were not supported for TO, just affinity. The template is not there either. But basically, you need to move the affinity from topicOperator to to entityOperator pod template.